x-pack/plugins/security_solution/public/management/components/endpoint_responder/components/platforms/platform_icon.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiIcon, type EuiIconProps } from '@elastic/eui';
+import React, { memo, useMemo } from 'react';
+import linuxSvg from './logos/linux.svg';
+import windowsSvg from './logos/windows.svg';
+import macosSvg from './logos/macos.svg';
+
+const getPlatformIcon = (platform: string) => {
+  switch (platform) {
+    case 'darwin':
+      return macosSvg;
+    case 'linux':
+      return linuxSvg;
+    case 'windows':
+      return windowsSvg;
+    default:
+      return `${platform}`;
+  }
+};
+
+export const PlatformIcon = memo<{
+  platform: string;
```

Comment:
Suggestion: can you type this to be a specific value? like:

```typescript
platform: 'darwin' | 'linux' | 'windows'
```

Reply:
8712a36c8b404d9cee17c41ae7829b3b4999aa58